inference-engine/src/vpu/myriad_plugin/myriad_executable_network.cpp

```diff
@@ -15,6 +15,8 @@
 #include <legacy/net_pass.h>
 #include <vpu/compile_env.hpp>
 
+#define IMPORTED_NETWORK_NAME "__importedExecutableNetworkFromBlobName"
```

Comment:
Please, use something like `static const char importedNetworkName[] = ...` or `constexpr auto importedNetworkName = ...` here as more preferable according to the code style. And be careful with comparing method at the line 190